cutil/cutil.go

```diff
@@ -51,13 +52,26 @@ func FirstN(n int, ss []string) []string {
 }
 
 func SecureRandomString() (string, error) {
-	bytes := make([]byte, 64)
 
+	random := mrand.New(mrand.NewSource(int64(time.Now().Nanosecond())))
+	
+	randStr := ""
+	randStr += string(rune(random.Intn(10) + 48))
+	randStr += string(rune(random.Intn(26) + 65))
+	randStr += string(rune(random.Intn(26) + 97))
```

Comment:
Could you please add comments in the code? Like:
 // generate a random number
  // generate a random upper case letter 
  // shuffle the string 
  etc.